chrome/content/zotero/elements/tagsBox.js

```diff
@@ -414,16 +414,16 @@
 
 			var itemID = this._item.id;
 
-			var t = document.createElement(rows > 1 ? 'textarea' : 'input', { is: 'shadow-autocomplete-input' });
+			var t = document.createElement(multiline ? 'textarea' : 'input', { is: 'shadow-autocomplete-input' });
 			t.setAttribute('class', 'editable');
 			t.setAttribute('value', value);
 			t.setAttribute('fieldname', fieldName);
 			t.setAttribute('ztabindex', tabindex);
 			t.setAttribute('ignoreblurwhilesearching', 'true');
 			t.setAttribute('autocompletepopup', 'PopupAutoComplete');
 			// Multi-line
-			if (rows > 1) {
-				t.setAttribute('rows', rows);
+			if (multiline) {
+				t.setAttribute('rows', 6);
```

Comment:
What's 6? Better to put magic numbers in a separate variable (and this can be `const`) for clarity.

Does this change any behavior?

Reply:
6 is the number of rows the multiline field has. It 100% should be a const variable for readability.

Before, the number of rows came either from the [number of new lines on paste](https://github.com/zotero/zotero/blob/794e89d307f62d5073e8bc08535fe7e85895992f/chrome/content/zotero/elements/tagsBox.js#L583) or it was [hardcoded here](https://github.com/zotero/zotero/blob/794e89d307f62d5073e8bc08535fe7e85895992f/chrome/content/zotero/elements/tagsBox.js#L479), which is the number of rows that would be used on shift-enter. 

I noticed that the fact that the number of rows of the textfield varies can break things - the multiline field can overlap with tags below of leave too much space before them. There's a few screenshots of this happening [here](https://github.com/zotero/zotero/issues/3334#issuecomment-1684343408). I thought the number of rows the multiline field occupies should always be the same to avoid this.